README.md

```diff
@@ -14,7 +14,7 @@ licence restrictions on an ongoing basis.
 If you already have Go installed, the easiest way of installing is with `go get`:
 
 ```
-go get github.com/sky-uk/licence-compliance-checker
+go get github.com/sky-uk/licence-compliance-checker/...
```

Comment:
Could we also mention that the binary is available as a GitHub asset and stored against each release tag

Reply:
Ignore this, I missread the changeset